xibabel/__init__.py

```diff
@@ -3,6 +3,7 @@
 import pathlib
 
 import nibabel as nib
+from nipy.algorithms.diagnostics.timediff import time_slice_diffs
```

Comment:
What do you think - should this be part of xibabel's interface?

Reply:
not sure what you mean

Reply:
I mean - by importing in `__init__.py` you make it part of the public interface.

Reply:
oh I see, then "no,  it should not be part of the public interface", I am just trying to get something going and figured we would probably even move most of this out of `__init__.py`,  just started sketching this out in what seemed like the most direct way.

Reply:
but I guess if we were to leave it here for now, would you prefer we import it as `_time_slice_diffs` or should we explicitly make an `__all__` with things we want to be officially public, or hide the import inside the function that uses it?

Reply:
Oh - it's fine - sorry - I was being anal.